File: SelectionSort.js
const selectionSort = (arr) => {
  // type your code here
  return arr.sort((a, b) => (a > b ? 1 : -1));
}




  // add your own tests in here
  console.log("Expecting: [-1, 2, 3, 5]");
  console.log("Result is =>", selectionSort([3, -1, 5, 2]));

  console.log("");

File: selectionSort.py
def selection_sort(arr) :
    for i in range(len(arr)):
        smallest = i
        for j in range(i + 1, len(arr)) :
            if arr[j] < arr[smallest] :
                smallest = j
        arr[j], arr[smallest] = arr[smallest], arr[j]
    return arr

print(selection_sort([19,34,17,10,22]))

